dbms/tests/queries/0_stateless/01030_limit_by_with_ties_error.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Simple .sql test with `-- { serverError xx }` is enough. Look at the examples in other tests.

Reply:
As I told you, linux error codes are from 0 to 255 and it is impossible to catch error number 490 from client.